src/index.jsx

```diff
@@ -5,6 +5,6 @@ export default (
   ComposedComponent,
   {
     LoadingIndicator = TailSpin,
-    ...rest,
```

Comment:
This is the airbnb linter.
...rest should be the last parameter, as this is the rest operator, it can't be somehting after that parameter. So the trailing comma is useless.